FordLabs/peoplemover
# Consolidate people tests

## Issue
Resolves N/A

## What was done
- [x] Moved all tests out of People.test.tsx that tested the PersonForm into the PersonForm.test.tsx file
- [x] Ensure all tests in People.test.tsx that rendered the whole app were tested in cypress or other unit tests and deleted the file

## How to test
1. Ensure that all tests that were in People.test.tsx that tested the PersonForm are now in PersonForm.test.tsx
2. Ensure that all things tested in People.test.tsx that required rendering the entire app are now only tested in cypress.
